logic/src/commonTest/kotlin/com/wire/kalium/logic/data/featureConfig/FeatureConfigRepositoryTest.kt

```diff
@@ -0,0 +1,80 @@
+package com.wire.kalium.logic.data.featureConfig
+
+import com.wire.kalium.logic.NetworkFailure
+import com.wire.kalium.logic.functional.Either
+import com.wire.kalium.logic.util.shouldFail
+import com.wire.kalium.logic.util.shouldSucceed
+import com.wire.kalium.network.api.ErrorResponse
+import com.wire.kalium.network.exceptions.KaliumException
+import io.mockative.Mock
+import io.mockative.classOf
+import io.mockative.given
+import io.mockative.mock
+import io.mockative.once
+import io.mockative.verify
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runTest
+import kotlin.test.Test
+
+@OptIn(ExperimentalCoroutinesApi::class)
+class FeatureConfigRepositoryTest {
+
+    @Test
+    fun whenFileSharingFeatureConfigSuccess_thenTheSuccessIsReturned() = runTest {
+        val arrangement = Arrangement()
+        arrangement.withSuccessfulResponse()
+
+        val actual = arrangement.featureConfigRepository.getFileSharingFeatureConfig()
+
+        actual.shouldSucceed { arrangement.expectedSuccess.value }
+        verify(arrangement.featureConfigRepository)
+            .coroutine { arrangement.featureConfigRepository.getFileSharingFeatureConfig() }
```

Comment:
This is rather a useless test as we are verifying what we actually invoked 😅  I think we should instead verify the api was called.